### PR TITLE
feat(config): expose JVB API port

### PR DESCRIPTION
**Pull request by NullIsNot0**
I had to expose JVB API port to read Colibri stats (http://jitsi.example.com:8080/colibri/stats) from remote monitoring system. Added new config variable: `JVB_TCP_MAPPED_API_PORT`

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -197,6 +197,7 @@ services:
         ports:
             - '${JVB_PORT}:${JVB_PORT}/udp'
             - '${JVB_TCP_PORT}:${JVB_TCP_PORT}'
+            - '${JVB_TCP_MAPPED_API_PORT}:8080'
```
Comment (saghul) [inline docker-compose.yml:200]: Can you rename this to `JVB_API_PORT`?

**Review by saghul — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (NullIsNot0) [reply docker-compose.yml]: Done

**Comment by Lithimlin** [comment]
Any updates on this? It would be a really helpful addition

**Comment by aaronkvanmeerten** [comment]
It appears this is now done via JVB_COLIBRI_PORT, so I'm closing this one.  Feel free to re-open if you disagree and want to discuss further.

**Pull request closed without merging**